Dockerfile

```diff
@@ -1,5 +1,12 @@
-# Our First stage, that builds the application
-FROM helsinkitest/node:12-slim AS react-builder
+# ==========================================
+FROM registry.access.redhat.com/ubi8/nodejs-16 AS deployable
```

Comment:
Should try to favour at least Node.js 18 if possible, since it's currently oldest LTS version.

Reply:
Very valid point but I struggled for half a day with upgrading to 18 without any meaningful progress and followed the instruction of minimal work when downgrading to 16 fixed all the issues that are coming on upgrade to 18.

Reply:
Could be ubi9 instead of ubi8

Reply:
ubi9 version of the image fails to build
Error: error:0308010C:digital envelope routines::unsupported       

As this was asked to be least effort I'll not resolve this right now.